### PR TITLE
Need to add more to README

### PR DESCRIPTION
**Pull request by rwxrob**
Here are some things we need to do

- [x] Add a word
- [x] Something else

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # foo
 
-Sample foo
+Sample foo here
```
Comment (rwxrob) [inline README.md]: Looks awesome.

**Review by ivanlemeshev — APPROVED** [review]
LGTM

**Review by ivanlemeshev — APPROVED** [review]

**Review by rwxrob — COMMENTED** [review]
adf

**Comment by rayct** [comment]
Reviewed

**Review by ivanlemeshev — APPROVED** [review]

**Review by rayct — COMMENTED** [review]
Nice!

**Pull request merged**